p2p/exchange.go

```diff
@@ -21,10 +21,15 @@ import (
 
 var log = logging.Logger("header/p2p")
 
-// the minimum number of headers of the same height received from trusted peers
-// to determine the network head. If all trusted header will return headers with
-// non-equal height, then the highest header will be chosen.
-const minTrustedHeadResponses = 2
+// minHeadResponses is the minimum number of headers of the same height
+// received from peers to determine the network head. If all trusted peers
+// will return headers with non-equal height, then the highest header will be
+// chosen.
+const minHeadResponses = 2
```

Comment:
I wonder if this value should be higher if untrusted peers are used. Currently if 2 peers out of 4 agree on header hash it will be taken as correct header. I think it should be stricter than that.

Reply:
I don't think that's necessary as tracked peers are randomised - but we can open an issue to discuss it since it's a param-related question!

Reply:
Lets open an issue then to discuss it there. My concern is that It is not random, but rather unpredictable given it uses range from map keys. Technically there is no guarantee of order, but there is also no guarantee for randomness.

Reply:
Created an issue #60